vcam-util.c

```diff
@@ -20,14 +20,16 @@ const struct option long_options[] = {
     {NULL, 0, NULL, 0}};
 
 const char *help =
-    " -h --help                    Print this informations.\n"
-    " -c --create                  Create a new device.\n"
-    " -m --modify  idx             Modify a device.\n"
-    " -r --remove  idx             Remove a device.\n"
-    " -l --list                    List devices.\n"
-    " -s --size    WIDTHxHEIGHT    Specify resolution.\n"
-    " -p --pixfmt  pix_fmt         Specify pixel format (rgb24,yuyv).\n"
-    " -d --device  /dev/*          Control device node.\n";
+    " -h --help                            Print this informations.\n"
+    " -c --create                          Create a new device.\n"
+    " -m --modify  idx                     Modify a device.\n"
+    " -r --remove  idx                     Remove a device.\n"
+    " -l --list                            List devices.\n"
+    " -s --size    WIDTHxHEIGHTxCROPRATIO  Specify virtual resolution or crop "
+    "ratio.\n"
+    "                                      (WxH, CR.n/CR.d, WxHxCR.n/CR.d)\n"
```

Comment:
Can you add more verbose messages?

Reply:
I have updated the help messages, please check whether it is clear enough? Thanks!

Reply:
I guess it still supports format WxH, CR.n/CR.d which needs a comment.

Reply:
@WayneLin1992  I have changed the `-s` help message as follow:
```
For instance:
   WxH:    640x480      Specify the virtual resolution.
   CR:     5/6          Apply crop ratio to the current resolution.
   WxHxCR: 640x480x5/6  Specify the virtual resolution apply with crop ratio.
```
I think the new messages have cover the semantics.